react/week2/my-app/src/TodoList.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import "./App.css";
+
+class TodoList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [
+        {
+          id: 1,
+          description: "Get out of bed",
+        },
+        {
+          id: 2,
+          description: "Brush teeth",
+        },
+        {
+          id: 3,
+          description: "Eat breakfast",
+        },
+      ],
+      check: false,
+    };
+  }
+
+  addTodo = () => {
+    const newTodo = [{ id: 4, description: "sleep well" }];
+    this.setState({ todos: this.state.todos.concat(newTodo) });
+  };
+  deleteTodo = (index) => {
```

Comment:
Here in deleteTodo , we can use filter option instead of slice. Also pass id as an arguments. Check below code : 
 
 deleteTodo = (id) => {
    let todos = Object.assign([], this.state.todos);
    todos= todos.filter(todo => todo.id !== id);
    this.setState({ todos: todos });
  };